CHANGELOG.md

```diff
@@ -2,6 +2,8 @@
 
 ## Version 2.5.2 (Under development)
 
+* **[Fix]** Fix warnings in when make authTokenHistory operations.
```

Comment:
`authTokenHistory` is an implementation detail of the SDK. It would be better if we provide a more meaningful description of the bug we fixed in the changelog.

Maybe something along the lines of:
```
* **[Fix]** Remove confusing warnings about "Failed to retrieve history state or none was found" when it doesn't make sense.  
```